docs/slides/day3.md

```diff
@@ -0,0 +1,272 @@
+class: middle, center
+
+# JumpStart Live
+In classroom conversation for day 3
+
+---
+
+# Software Development Process
+
+![Software Dev Process meme](day3_images/software-dev-process.jpg)
+
+---
+
+# Warmup
+
+Share what you learned from our last session about conditionals (if, else if, else…)
+
+Have a classmate code review your election time code (and vice versa).
+
+* Is it clear what the program is doing?
+* Are there sufficient comments?
+* Does the control flow of the program make sense?
+* Does the code follow the ruby style guide?
+* Is there repeated code anywhere?
+* Are edge cases appropriately handled?
+
+Focus on using programming vocabulary (e.g, blocks, loops, iterator, variable assignment, conditional statement) and be as precise as possible when reviewing.
+
+---
+
+# Loops
+
+Code that can be repeated more than once
+
+What's the difference between the following types of loops?
+
+* Sentinel Controlled
+* Counter Controlled
+* Infinite
+
+
+---
+
+# Sentinel Controlled
+
+.left-column[
+    While Loop
+
+```ruby
+while <something is true>
+  # do stuff
+end
+```
+]
+
+.right-column[
+    Until Loop
+
+```ruby
+until <something is false>
+  # do stuff
+end
+```
+]
+
+--
+
+## When would you use an `until` instead of a `while`?
+
+---
+
+# Convert to an until loop
+
+.left-column[
+    ```ruby
+    puts "Hello we're going to total 
+      some numbers!"
+    puts "Enter a negative number 
+      to quit."
+
+    total = 0
+    input = gets.chomp.to_i
+
+    while input > -1
+      total += input
+      input = gets.chomp.to_i
+    end
+
+    puts "Total is #{total}"
+    ```
+]
+
```

Comment:
If you want there to be a pause between showing the left-column and the right-column in slide 7, put a `--` line on this line between `]` and `.right-column[`